diagnostic.md

```diff
@@ -140,7 +166,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+5,4,3,2,1 from left to right on one line
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).